include/rocksdb/utilities/backupable_db.h

```diff
@@ -396,9 +403,12 @@ class BackupEngine {
   }
 
   // checks that each file exists and that the size of the file matches our
-  // expectations. it does not check file checksum.
+  // expectations if with_checksum is false
+  // inspects whether the file checksum matches our expectation if
+  // with_checksum is true
   // Returns Status::OK() if all checks are good
-  virtual Status VerifyBackup(BackupID backup_id) = 0;
```

Comment:
@pdillinger I'm not sure, is possible some users developed their own backup engine which is inherited from class BackupEngine?

Reply:
Ha. Good question. IMHO, that's rare enough that any affected users can modify their code. We have this listed under "public API change" so that seems like fair warning. :)